project/npda/forms/visit_form.py

```diff
@@ -61,15 +58,7 @@ class Meta:
             "hospital_discharge_date",
             "hospital_admission_reason",
             "dka_additional_therapies",
-            "hospital_admission_other",
-            # calculated fields
-            "height_centile",
```

Comment:
As these are calculated fields it should not be possible to submit them through the form